views/private_subnet/show.erb

```diff
@@ -96,7 +96,10 @@
           <tr>
             <td class="whitespace nowrap py-4 pl-4 pr-3 text-sm font-medium text-gray-900 sm:pl-6" scope="row">
               <% if Authorization.has_permission?(@current_user.id, "Firewall:view", fw[:id]) %>
-                <a href="<%= "#{@project_data[:path]}/firewall/#{fw[:id]}" %>" class="text-orange-600 hover:text-orange-700"><%= fw[:name] %></a>
+                <a
+                  href="<%= "#{@project_data[:path]}/location/#{fw[:location]}/firewall/#{fw[:name]}" %>"
```

Comment:
Its serializer has `include_path` option. We can use `fw[:path]` for the second part